vaadin-dialog-flow-parent/vaadin-dialog-flow/src/main/java/com/vaadin/flow/component/dialog/Dialog.java

```diff
@@ -995,4 +995,36 @@ public Style getStyle() {
                 "Dialog does not support adding styles to overlay");
     }
 
+    /**
+     * Close all the Dialog components associated within the current UI.
+     */
+    public static void closeAllDialogs() {
```

Comment:
It's good practice to always provide an overload that takes and explicit UI instance and then a shorthand that uses `UI.getCurrent()` rather than always only using `getCurrent()` implicitly.